README.md

```diff
@@ -19,9 +19,9 @@ Find cells with different criteria, and intersect resulting sets:
 >>> client = Client('https://cells.dev.hubmapconsortium.org/api/')
 
 >>> [m for m in dir(client) if m.startswith('select_')]
-['select_cells', 'select_clusters', 'select_genes', 'select_organs']
+['select_cells', 'select_clusters', 'select_datasets', 'select_genes', 'select_organs']
 
->>> cells_with_vim = client.select_cells(where='gene', has=['VIM > 0.5'], genomic_modality='rna')
+>>> cells_with_vim = client.select_cells(where='gene', has=['VIM > 0.5'], genomic_modality='rna', logical_operator='and')
```

Comment:
Is `logical_operator` required? With only one `has` it seems unnecessary, and should it be a default kwarg?

Reply:
Logical operator is required for certain types of queries (gene->cell, cluster/organ->gene, gene->organ/cluster).  You're right.  It is unnecessary for an input set of 1.  I'll fix that client-side.